ebook/es/content/014-creating-custom-bash-commands.md

```diff
@@ -0,0 +1,83 @@
+# Creating custom bash commands

```

Comment:
Thank you for this PR! Seems like this file was added by mistake. Would you mind removing it whenever you have a chance?

Reply:
Yes, I'm sorry, I have the doubt about the file name to leave in english as other files in folder standing there before my commit or translate it to, and I left the original file in english at the folder by mistake for this.
I will continue with next PR :-)